src/AstRunner/Resolver/AnnotationDependencyResolver.php

```diff
@@ -31,6 +31,20 @@ public function __construct(TypeResolver $typeResolver)
 
     public function processNode(Node $node, ClassReferenceBuilder $classReferenceBuilder, TypeScope $typeScope): void
     {
+        if ($node instanceof Node\Stmt\ClassLike) {
```

Comment:
Most parts are duplicated logic from reading the method doc blocks below. I will clean this up later, but in any case it will make this resolver more unwieldy. Should I separate reading class/method annotations into separate resolvers instead?